doc/source/examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability.rst

```diff
@@ -64,15 +65,14 @@ Parameter File
 Simulation Control
 ~~~~~~~~~~~~~~~~~~
 
-Time integration is handled by a 1st order backward differentiation scheme 
-(`bdf1`), for a :math:`0.75` s simulation time with an initial 
+Time integration is handled by a 2nd order backward differentiation scheme
+(``bdf2``), for a :math:`0.75\, \text{s}` simulation time with an initial
 time step of :math:`0.0002` seconds. Time-step adaptation is enabled using ``adapt=true``
-and the max CFL is :math:`0.9`.
+and the max CFL is :math:`0.8`.
 
 .. note::   
     This example uses an adaptive time-stepping method, where the 
-    time-step is modified during the simulation to keep the maximum value of the CFL condition below a given threshold (0.9 here). Using ``output control = time``, and ``output time = 0.005`` the simulation results are written every 0.005 s.
-
+    time-step is modified during the simulation to keep the maximum value of the CFL condition below a given threshold (:math:`0.9` here). Using ``output control = time``, and ``output time = 0.005`` the simulation results are written every :math:`0.005\, \text{s}`.
 
 .. code-block:: text
 
```

Comment:
At line 94 Replace : The ``multiphysics`` subsection enables to turn on ``true`` and off ``false`` the physics of interest. Here ``VOF`` and ``fluid dynamics`` are chosen (``fluid dynamics`` is true by default).

by : 

The ``multiphysics`` subsection enables to turn on (``true``) and off (``false``) the physics of interest. Here ``VOF`` and ``fluid dynamics`` are chosen (``fluid dynamics`` is ``true`` by default).